### ISSUE TITLE
Using `build.sourcemap` in Vite config causes error during the server files build step

### ISSUE THREAD

**Issue by spacedawwwg** [brillout/vite-plugin-ssr#612]
### Description

Using `build.sourcemap` (https://vitejs.dev/config/build-options.html#build-sourcemap) causes the the following error with vite-plugin-ssr v0.4.71

### Error Message + Error Stack

```shell
vite v4.0.4 building SSR bundle for production...
✓ 1062 modules transformed.
rendering chunks (93)...[@brillout/vite-plugin-import-build:vite-plugin-ssr] [@brillout/vite-plugin-import-build][Bug] You stumbled upon a bug in the source code of @brillout/vite-plugin-import-build. Reach out at https://github.com/brillout/vite-plugin-import-build/issues/new and include this error stack (the error stack is usually enough to fix the problem).
Error: [@brillout/vite-plugin-import-build][Bug] You stumbled upon a bug in the source code of @brillout/vite-plugin-import-build. Reach out at https://github.com/brillout/vite-plugin-import-build/issues/new and include this error stack (the error stack is usually enough to fix the problem).
    at assert (/Users/project/node_modules/@brillout/vite-plugin-import-build/dist/utils/assert.js:10:11)
    at findBuildEntry (/Users/project/node_modules/@brillout/vite-plugin-import-build/dist/plugin/findBuildEntry.js:11:32)
    at findBuildEntry (/Users/project/node_modules/@brillout/vite-plugin-import-build/dist/plugin/index.js:64:80)
    at getImporterCode (/Users/project/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/importBuild/index.js:22:40)
    at /Users/project/node_modules/@brillout/vite-plugin-import-build/dist/plugin/index.js:63:43
    at Array.map (<anonymous>)
    at generateImporter (/Users/project/node_modules/@brillout/vite-plugin-import-build/dist/plugin/index.js:63:14)
    at Object.generateBundle (/Users/project/node_modules/@brillout/vite-plugin-import-build/dist/plugin/index.js:27:13)
    at file:///Users/project/node_modules/vite/node_modules/rollup/dist/es/shared/rollup.js:23766:40
    at runMicrotasks (<anonymous>) {
  code: 'PLUGIN_ERROR',
  plugin: '@brillout/vite-plugin-import-build:vite-plugin-ssr',
  hook: 'generateBundle'
}
```

**Comment by brillout** [comment]
Fix released at `0.4.71-draft.17337`.

**Comment by spacedawwwg** [comment]
I'll keep an eye out for the 0.4.72 bump @brillout 👍🏻
